cache/http.go

```diff
@@ -16,7 +16,7 @@ import (
 	"sync"
 )
 
-var blobNameSHA256 = regexp.MustCompile("^/?(ac/|cas/)([a-f0-9]{64})$")
+var blobNameSHA256 = regexp.MustCompile("^/?(.*/)?(ac/|cas/)([a-f0-9]{64})$")
```

Comment:
Very egoistic reasons. I have a server running under `/cache`, and wanted to keep the same behavior. I also had the tests written already, so I left this in. I think I will implement multiple cache directories like we said in https://github.com/buchgr/bazel-remote/issues/5. Do you object to leaving this in?

Reply:
With multiple cache directories meaning localhost:8080/windows-cache localhost:8080/linux-cache being distinct caches?

Reply:
In this PR, it will accept both URLs but store the files in the same directory. In a follow-up, we can then store the AC files in different folders (no need to split the CAS IMO).